src/index.android.js

```diff
@@ -0,0 +1,41 @@
+// On Android, both Image and Photo view are supported,
+// and defaultSource is converted to an uri that the
```

Comment:
I think it should be "a uri", but not a big thing (I can fix it after the merge)

Reply:
I think you're right :)